tests/conftest.py

```diff
@@ -74,6 +75,7 @@ def pytest_configure():
             'rest_framework_gis',
             'rest_framework.authtoken',
             'crispy_forms',
+            'stored_messages',
```

Comment:
Huh, actually I don't need it any more. Needed it when I revived the job_progress tests in b23c04284339a42718f6e70ebc599be8244b1b89, but half of those tests are skipped right now. I'll remove it for now. We'll probably need it again when re-reviving those tests (or replacing them with new ones), but the error message if we forget to re-add the app will be clear enough, IIRC.